composer.json

```diff
@@ -126,6 +126,7 @@
         "symfony/security-core": "4.2.* <4.2.3",
         "symfony/swiftmailer-bundle": "2.6.* <2.6.2",
         "symfony/twig-bundle": "4.1.0",
+        "terminal42/contao-ce-access": "<3.0",
```

Comment:
why is this not `*`?

Reply:
Because you might release a version 3 that is compatible with Contao again. 🤷‍♂